src/components/settings/EditOrganizationInformationsDialog.tsx

```diff
@@ -0,0 +1,323 @@
+              <Button variant="secondary" onClick={() => hiddenFileInputRef?.current?.click()}>
+                {translate('text_62ab2d0396dd6b0361614d18')}
+              </Button>
+              <Typography variant="caption" color={logoUploadError ? 'red' : ''}>
+                {translate('text_62ab2d0396dd6b0361614d20')}
+              </Typography>
+            </AvatarUploadWrapper>
+            <input
+              type="file"
+              accept="image/png, image/jpeg"
+              ref={hiddenFileInputRef}
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                setLogoUploadError(false)
+                const file = event?.target?.files?.[0]
+
+                if (file) {
+                  getBase64(file)
+                }
+              }}
+              style={{ display: 'none' }}
+            />
+          </AvatarContainer>
+        </FormSection>
+        <FormSection>
+          <TextInput
+            name="name"
+            label={translate('text_62ab2d0396dd6b0361614d5c')}
+            placeholder={translate('text_62ab2d0396dd6b0361614d5c')}
+            value={organization?.name || ''}
+            disabled={true}
+          />
+        </FormSection>
+        <FormSection>
+          <TextInputField
+            name="legalName"
+            label={translate('text_62ab2d0396dd6b0361614d6c')}
+            placeholder={translate('text_62ab2d0396dd6b0361614d6c')}
+            // eslint-disable-next-line jsx-a11y/no-autofocus
+            autoFocus
+            formikProps={formikProps}
+          />
+        </FormSection>
+        <FormSection>
+          <TextInputField
+            name="legalNumber"
+            label={translate('text_62ab2d0396dd6b0361614d7c')}
+            placeholder={translate('text_62ab2d0396dd6b0361614d7c')}
+            // eslint-disable-next-line jsx-a11y/no-autofocus
+            autoFocus
+            formikProps={formikProps}
+          />
+        </FormSection>
+        <FormSection>
+          <TextInputField
+            name="email"
+            label={translate('text_62ab2d0396dd6b0361614d8c')}
+            placeholder={translate('text_62ab2d0396dd6b0361614d8c')}
+            // eslint-disable-next-line jsx-a11y/no-autofocus
+            formikProps={formikProps}
+          />
+        </FormSection>
+        <AddressSection>
+          <TextInput
+            name="addressLine1"
+            label={translate('text_62ab2d0396dd6b0361614d70')}
+            placeholder={translate('text_62ab2d0396dd6b0361614d78')}
+            value={organization?.addressLine1 || ''}
+            onChange={(value) => formikProps.setFieldValue('addressLine1', value)}
+          />
+          <TextInput
+            name="addressLine2"
+            placeholder={translate('text_62ab2d0396dd6b0361614d80')}
+            value={organization?.addressLine2 || ''}
+            onChange={(value) => formikProps.setFieldValue('addressLine2', value)}
```

Comment:
Sorry this is not how we use formik works

You need to use the component suffixed by `Field` to use it with Formik (you did it right on the `email` field

(I don't comment all of them but it works for all the other one on this component except for the `name`

Reply:
I thought it was the only way to have an input without a label easilly but it's not mandatory ˆˆ

Are you ok to still use it for the name, as it' not handled by formik ?

Reply:
Yep, for the name it make sense :)